moveit_ros/moveit_servo/src/servo_parameters.yaml

```diff
@@ -0,0 +1,353 @@
+servo:
+################################ ROBOT SPECIFIC CONFIG #############################
+  move_group_name: {
+    type: string,
+    default_value: "panda_arm",
+    description: "The name of the moveit move_group of your robot"
+  }
+
+  planning_frame: {
+    type: string,
+    default_value: "panda_link0",
+    description: "The name of moveit planning frame."
+  }
+
+  ee_frame_name: {
+    type: string,
+    default_value: "panda_link8",
+    description: "The name of end-effector frame of your robot"
+  }
+
+  robot_link_command_frame: {
+    type: string,
+    default_value: "panda_link0",
+    description: "The frame of incoming command, change this to your own value when using a different robot."
+  }
+
+  monitored_planning_scene_topic: {
+    type: string,
+    default_value: "/planning_scene",
+    description: "The name of the planning scene topic. \
+                  planning_scene_monitor::PlanningSceneMonitor::DEFAULT_PLANNING_SCENE_TOPIC"
+  }
+
+################################ OUTPUTS #########################################
+  status_topic: {
+    type: string,
+    default_value: "~/status",
+    description: "The topic to which the status will be published"
+  }
+
+  command_out_topic: {
+    type: string,
+    default_value: "/panda_arm_controller/joint_trajectory",
+    description: "The topic on which servo will publish the joint trajectory \
+                  Change this to the topic your controller requires."
+  }
+
+  command_out_type: {
+    type: string,
+    default_value: "trajectory_msgs/JointTrajectory",
+    description: "The type of command that servo will publish",
+    validation: {
+      one_of<>: [["trajectory_msgs/JointTrajectory", "std_msgs/Float64MultiArray"]]
+    }
+  }
+
+  num_outgoing_halt_msgs_to_publish: {
```

Comment:
I believe this parameter actually isn't being used anymore. Please do `grep -r num_outgoing_halt_msgs_to_publish` then delete any references to it.